### DISCUSSION TITLE
🔔 Introducing JointJS+ v3.7

### DISCUSSION THREAD

**Discussion by marekhozak10** [clientIO/joint#2197] [News]
Introducing JointJS+ v3.7, bringing you more demo applications, increased stability and even greater flexibility 🔝

The latest version of our library significantly improves support for foreign objects, offers lots of great demo apps like Activity or Sankey diagram, and as always serves as a reliable code base on your way to fast and confident development!

Want to learn more about JointJS+ v3.7? Read about the key highlights ➡️ [https://www.jointjs.com/blog/new-release-jointjs-v3-7](https://www.jointjs.com/blog/new-release-jointjs-v3-7)

----

**TLDR:**
🤝 Improved support for foreign objects (HTML)
ထ Unlimited customization options thanks to the preventDefaultInteraction function
🆕 New API to initiate Stencil dragging
🪄 Dynamic input values
🖐️ Enabling direct interaction with selected elements
👌 New demo applications, including Sankey diagram, Activity diagram and more
🌊 Bug fixes before the big wave
